transcription/sphinx.go

```diff
@@ -0,0 +1,57 @@
+// Package transcription implements functions for the manipulation and
+// transcription of audio files.
+package transcription
+
+import (
+	"bufio"
+	"encoding/json"
+	"os"
+	"os/exec"
+)
+
+//Transcription contains the transcription text and metadata of a transcription
+//job
+type Transcription struct {
+	TextTranscription string
+	Metadata          string
+}
+
+// SphinxTranscription transcribes a given file using Sphinx.
+// File name should not include the type extension.
+func SphinxTranscription(fileName string) (Transcription, error) {
+	var result Transcription
```

Comment:
Nit: this function could probably use one or two spaces to make it easier on the eyes